samples/zigbee/light_switch/sysbuild_fota.conf

```diff
@@ -4,6 +4,5 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-SB_CONFIG_DFU_MULTI_IMAGE_PACKAGE_BUILD=y
+SB_CONFIG_PARTITION_MANAGER=y
```

Comment:
I don't get why you would be enabling this when it should be enabled by default

Reply:
Right, enabling PM was redundant here. I've removed it